h2o-docs/src/product/automl.rst

```diff
@@ -68,9 +68,14 @@ Optional Miscellaneous Parameters
 
 - `max_runtime_secs_per_model <data-science/algo-params/max_runtime_secs_per_model.html>`__: Specify the max amount of time dedicated to the training of each individual model in the AutoML run. Defaults to 0 (disabled). Note that setting this parameter can affect AutoML reproducibility.
 
+- **modeling_plan**: The list of modeling steps to be used by the AutoML engine (they may not all get executed, depending on other constraints).
```

Comment:
- **modeling_plan**: The list of modeling steps to be used by the AutoML engine. (They may not all get executed, depending on other constraints.)